nni/experiment/config/training_services/remote.py

```diff
@@ -64,11 +64,14 @@ def _validate_canonical(self):
 class RemoteConfig(TrainingServiceConfig):
     platform: Literal['remote'] = 'remote'
     machine_list: List[RemoteMachineConfig]
-    reuse_mode: bool = True
+    reuse_mode: bool = False
```

Comment:
Set it to `True` will choose remote v2, which supports `shared_storage` that v3 does not (yet).

Reply:
So no matter whether `reuse_mode` is set to true or false, it will reuse, right?

Looks a bit confusing. Maybe we can set this flag as deprecated?

Reply:
There is no real "non-reuse" for remote training service. For cluster based training services reuse means to keep the VMs alive, and for remote the workers are always alive.

I have thought about `version: 1 | 2 | 3` and later found it unnecessary.
Since v3 provides all of v1's capacity, I have decided to remove v1's entrance from `NNIManager.initTrainingService()` which has already been too complex. And then, `version: 2` equals to `reuse_mode: True` and `version: 3` equals to `reuse_mode: False`.

I consider this as a deprecated compatibility option, though not actually deprecated yet because v2 has extra features. When v3 is complete and stable I will remove v2's code as well as this option.
For a deprecated option, I think compatible is more import than "good", and `reuse_mode` provides better compatibility.